packages/wonder-blocks-dropdown/src/components/__tests__/single-select.test.js

```diff
@@ -1,111 +1,207 @@
 //@flow
 import * as React from "react";
-import IconButton from "@khanacademy/wonder-blocks-icon-button";
-import {mount} from "enzyme";
+import {render, screen} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom/extend-expect";
 
-import DropdownOpener from "../dropdown-opener.js";
-import SelectOpener from "../select-opener.js";
 import OptionItem from "../option-item.js";
 import SingleSelect from "../single-select.js";
-import {keyCodes} from "../../util/constants.js";
-import SearchTextInput from "../search-text-input.js";
 
 jest.mock("../dropdown-core-virtualized.js");
 
+const FRAME_DURATION = 17;
```

Comment:
Is there a specific reason why we use this value?

Reply:
Yes, it's approximately 1000/60 or 1/60th of a second.  It's not really necessary to use that value though since we're using fake timers.